.gitignore

```diff
@@ -14,9 +14,10 @@ log.txt
 *.dll
 *.pyc
 
-# DS_2_4 #Ignore files unneeded.
-# (i.e. cmake and /dep/... files)
-#*.cmake
+//! DS_2_4 #Ignore files unneeded.
+//! (i.e. cmake and /dep/... files)
+//!*.cmake
```

Comment:
Have these been commented out? Or is this some formatting for .gitignore I didn't know about?

Reply:
I added a comment that will hopefully be included in our doxygen documentation in the future. The comment is saying the that Developer Story DS 2.4 was resolved below. Does that make sense?